sda-commons-server-testing/README.md

```diff
@@ -51,6 +51,9 @@ YAML or JSON and ignores the order of keys. If possible, prefer the other varian
 content should always be reproducible. Note that the [AsyncAPI](../sda-commons-shared-asyncapi) and
 [OpenAPI](../sda-commons-server-openapi) generations export reproducible content. 
 
+
+# JUnit 4
```

Comment:
I think level 1 heading for not used intentionally. Could you just adjust the following heading? I.e. "Provided JUnit 4 rules" and later on "Provided JUnit 5 extensions"?